apps/tandoor/config.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "../schema.json",
+  "name": "Tandoor",
+  "port": 8341,
+  "available": true,
+  "exposable": true,
+  "id": "tandoor",
+  "tipi_version": 1,
+  "version": "1.5.4",
+  "categories": ["data"],
+  "description": "Drop your collection of links and notes. Get Tandoor and never look back onto a time without recipe management, storage, sharing and collaborative cooking!",
+  "short_desc": "Manage your ever growing recipe collection online.",
```

Comment:
Maybe a shorter short_desc like "Recipe collection manager." otherwise it will be truncated in apps view.